apps/src/music/views/GenerateCode.tsx

```diff
@@ -197,8 +205,9 @@ const GenerateCode: React.FunctionComponent<GenerateCodeProps> = ({
           {useAdlib && (
             <Adlib
               adlib={useAdlib}
-              onChange={(text, choices) => {
+              onChange={(text, choices, localized) => {
                 setPromptText(text);
+                setLocalizedPromptText(localized);
```

Comment:
Maintaining both types of strings means that the existing analytics event used elsewhere in this component would still report the English string to avoid metrics noise.